### PR TITLE
Protocol sources fixes

### PR DESCRIPTION
**Pull request by Persson-dev**
As mentioned in #5388, i moved the HANDLE_READ and HANDLE_PACKET_READ macros in Packetizer.h.
I also fixed some compiler warnings in the 1.14 protocol source file.

### CODE REVIEW

**Review by 12xx12 — APPROVED** [review]

File: src/Protocol/Protocol_1_14.cpp [hunk 1]
```diff
@@ -732,7 +719,9 @@ UInt8 cProtocol_1_14::GetProtocolEntityType(const cEntity & a_Entity) const
 				case PType::pkSplashPotion: return 81;
 				case PType::pkFirework: return 26;
 				case PType::pkWitherSkull: return 92;
+				default: break;
```
Comment (bearbin) [inline src/Protocol/Protocol_1_14.cpp:722]: Not sure about this change, this switch should be an all-cases-handled one so we get a warning when updating the PTypes. (The break on the line below is fine though).

Comment (12xx12) [reply src/Protocol/Protocol_1_14.cpp:722]: Oh yeah - I remember. That's why there are that many useless case statements in the monster section.

File: src/Protocol/Protocol_1_14.cpp [hunk 2]
```diff
@@ -733,6 +720,7 @@ UInt8 cProtocol_1_14::GetProtocolEntityType(const cEntity & a_Entity) const
 				case PType::pkFirework: return 26;
 				case PType::pkWitherSkull: return 92;
 			}
+			break;
```
Comment (12xx12) [inline src/Protocol/Protocol_1_14.cpp:723]: Shouldn't the ```break;``` statement be inside the ```{ }```

Comment (12xx12) [reply src/Protocol/Protocol_1_14.cpp:723]: Yeah - My bad. The GitHub web interface made it seem like the break was somewhere else.

**Pull request merged**